include/video.h

```diff
@@ -401,9 +401,17 @@ void GFX_EndUpdate( const uint16_t *changedLines );
 void GFX_LosingFocus();
 void GFX_RegenerateWindow(Section *sec);
 
+void GFX_RefreshTitle();
+void GFX_RefreshAnimatedTitle();
+void GFX_NotifyBooting();
+void GFX_SetAudioCaptureMark(const bool is_capturing);
+void GFX_SetVideoCaptureMark(const bool is_capturing);
+void GFX_SetProgramName(const std::string& segment_name,
+                        const std::string& canonical_name);
+void GFX_SetCycles(const int32_t cycles);
```

Comment:
Can we introduce some light decoupling here? For now, it's just mostly about renaming these functions, e.g., from ` GFX_SetAudioCaptureMark` to `GFX_NotifyAudioCaptureStatus`. Then `GFX_NotifyAudioCaptureStatus` would call another function called `GFX_SetTitlebarAudioCaptureMark`.

The reason I'm proposing this is because with the OSD stuff `GFX_NotifyAudioCaptureStatus` will then call a second method called `OSD_SetAudioCaptureMark` or something, and if you do it this way now, I won't need to rework it later 😄

Reply:
OK, changed.